src/app.rs

```diff
@@ -25,6 +25,13 @@ pub fn App() -> impl IntoView {
                     <Route path="/projects" view=Projects/>
                     <Route path="/photos" view=Photos/>
                     <Route path="/codecanvas" view=|| view! { <div>"CodeCanvas - Coming Soon"</div> }/>
+                    <Route path="/*any" view=|| view! {
```

Comment:
not totally sure this will work if /*any is special but f it let's see if it fixes